models/issue.go

```diff
@@ -1103,7 +1103,8 @@ type IssuesOptions struct {
 	UpdatedAfterUnix   int64
 	UpdatedBeforeUnix  int64
 	// prioritize issues from this repo
-	PriorityRepoID int64
+	PriorityRepoID       int64
+	ExcludeArchivedRepos util.OptionalBool
```

Comment:
I would propose `IsArchived` or `OwnedByArchivedRepos` -> looks better to me throu ... - if you have a better idear just write :)

Why I do not like ExcludeXY -> because it's not a optional filter option ... it can only exclude but not the oposite